x-pack/plugins/security_solution/public/common/lib/cell_actions/default_cell_actions.test.tsx

```diff
@@ -43,7 +43,7 @@ describe('default cell actions', () => {
           header: columnHeaders.find((h) => h.id === header.id),
           pageSize,
           scopeId: tableId,
-        });
+        }) as EuiDataGridColumnCellAction;
```

Comment:
Rather than using `as`, can we use `const buildAction = (dataTableCellAction: DataTableCellAction): EuiDataGridColumnCellAction =>` up above so that types are actually checked for issues instead of being coerced?

Reply:
This one's not resolvable without an `as` due to the way we recently changed EuiDataGrid's types. Disregard this comment